<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>1170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_contactList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>950</y>
      <width>155</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>연락처</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_chatList">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>950</y>
      <width>155</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>대화</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_groupChatList">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>950</y>
      <width>155</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>회의</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_notice">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>950</y>
      <width>155</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>공지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_serverOn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>서버 시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_serverOff">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>서버 종료</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_serverStatus">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>프로그램 종료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_logout">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>로그아웃</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
